CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CheckDrive.Web.Stores.Debts;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckDrive.Web.Controllers
+{
+    public class DebtsController : Controller
+    {
+        private readonly IDebtsStore _store;
+        public DebtsController(IDebtsStore debtsStore)
+        {
+            _store = debtsStore;
```

Comment:
should we check that is debtsStore not null and if debtsStore is null, return new ArgumentNullException

Reply:
fixed